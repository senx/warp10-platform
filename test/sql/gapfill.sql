-- This file and its contents are licensed under the Apache License 2.0.
-- Please see the included NOTICE for copyright information and
-- LICENSE-APACHE for a copy of the license.

\unset ECHO
\o /dev/null
\ir include/test_utils.sql
\o
\set ECHO queries
\set VERBOSITY default

\c :TEST_DBNAME :ROLE_SUPERUSER

SELECT allow_downgrade_to_apache();
SET timescaledb.license_key='ApacheOnly';

\set ON_ERROR_STOP 0
SELECT locf(1);
SELECT interpolate(1);
SELECT time_bucket_gapfill(1,1,1,1);
\set ON_ERROR_STOP 1
