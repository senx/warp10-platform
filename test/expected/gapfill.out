-- This file and its contents are licensed under the Apache License 2.0.
-- Please see the included NOTICE for copyright information and
-- LICENSE-APACHE for a copy of the license.
\unset ECHO
SELECT allow_downgrade_to_apache();
 allow_downgrade_to_apache 
---------------------------
 
(1 row)

SET timescaledb.license_key='ApacheOnly';
SELECT locf(1);
ERROR:  function "locf" is not supported under the current license "ApacheOnly"
HINT:  Upgrade to a Timescale-licensed binary to access this free community feature
SELECT interpolate(1);
ERROR:  function "interpolate" is not supported under the current license "ApacheOnly"
HINT:  Upgrade to a Timescale-licensed binary to access this free community feature
SELECT time_bucket_gapfill(1,1,1,1);
ERROR:  function "time_bucket_gapfill" is not supported under the current license "ApacheOnly"
HINT:  Upgrade to a Timescale-licensed binary to access this free community feature
